theano/tensor/tests/test_basic.py

```diff
@@ -5195,9 +5195,10 @@ def just_vals(v):
         # test broadcast flag for constant value of 1
         c = reshape(b, (b.shape[0], b.shape[1], 1))
         f = self.function([b], c)
+        g = theano.gof.FunctionGraph([b], [c])
         assert numpy.all(f(numpy.asarray([[0, 1, 2], [3, 4, 5]])) ==
                          numpy.asarray([[[0], [1], [2]], [[3], [4], [5]]]))
-        assert (f.maker.fgraph.toposort()[-2].outputs[0].type.broadcastable ==
+        assert (g.toposort()[-1].outputs[0].type.broadcastable ==
```

Comment:
Hi @lamblin ,
Does this fix seems fine to you?
I did this to handle when calling with ```mode=FAST_COMPILE```.